src/styles/index.scss

```diff
@@ -1,3 +1,118 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+@import "variables";
+@import "reset";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
```

Comment:
preferably to sort your css styles (everywhere)
for example:
- position
- display
- margin / padding
- font settings
- colors